scapy/utils.py

```diff
@@ -1359,7 +1359,7 @@ def get_terminal_width():
         else:
             return None
 
-def pretty_routes(rtlst, header, sortBy=0):
+def pretty_list(rtlst, header, sortBy=0):
```

Comment:
Why are you changing this name ?

Reply:
Because we're now using the function to pretty something other than routes (IFACES)

Reply:
I understand. Could you also change the docstring ? =)

Reply:
Updated, thanks